phaseshifts/wrappers.py

```
@@ -0,0 +1,752 @@
+        for elem in set(symbols):
+            at_file = os.path.join(output_dir, "at_%s.i" % elem)
+            if not os.path.isfile(at_file):
+                print("\nCalculating atomic charge density for %s..." % elem)
+                atomic_dict[elem] = atorb.Atorb.calculate_Q_density(
+                    element=elem, output_dir=output_dir
+                )
+            else:
+                atomic_dict[elem] = at_file
+        return atomic_dict
+
+    def autogen_from_input(  # noqa: MC0001
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `MC0001`)

Remove unused `noqa` directive

(RUF100)